### DISCUSSION TITLE
Z390 monterey

### DISCUSSION THREAD

**Discussion by BokitoNL38** [baughmann/designaire-z390-intel-i9-9900k-opencore#74] [Q&A]
Hi there,

First of all I would like to thank you for your great work, and sharing of the EFI folder's ! 

I am trying to help my uncle, for which I build a hackintosh a few years ago.
He is still running Mojave using Clover.

It's the following hardware : 
Z390 designare bios F6
Core i9 9900K
32GB Corsair 2666mhz dual channel DDR4
Crucial MX500 1TB Sata SSD
Nvidia GTX780 kepler GPU

Do we really need to update the bios to F9i , in order to make Monterey to work ? 
I am affraid to upgrade the bios, since we cannot perhaps downgrade anymore to F6 or making a situation where Mojave wont boot at all anymore...

Looking forward to your message, thanks in advance,

Regards,
Floris

**Comment by seven-of-eleven** [comment]
Hi @BokitoNL38 Sorry for the delayed response. You should be able to install with your existing bios. AFAIK the only BIOS version with issues is F9j. So as long as you avoid that you should be good. 

There can be benefits to having an up to date bios but if everything works for you then I wouldn't worry about it. You will have to manually set your bios settings but that's fairly easy if you follow the [OpenCore guide](https://dortania.github.io/OpenCore-Install-Guide/config.plist/coffee-lake.html#cleaning-up) recommendations.

Let me know if you run into any specific issues (open a new issue) and I'll try and help.

**Reply by seven-of-eleven** [reply]
I just saw your discussion in the other thread. Hopefully you've got it figured out. Best of luck.
